### PR TITLE
Add basic information on transfer permissions

### PR DESCRIPTION
**Pull request by sallain**
This adds a section on the permissions that need to be given to the
Archivematica user for files and directories, and a note about what it
looks like when files/directories do not have sufficient permissions.

Connected to archivematica/Issues#1019

### CODE REVIEW

File: user-manual/transfer/transfer.rst [hunk 1]
```diff
@@ -126,6 +127,21 @@ Archivematica instance that you are using. For more information on configuring
...
 Archivematica <scaling-archivematica>`.
 
+.. _transfer-permissions:
+
+Permissions
+^^^^^^^^^^^
+
+In order for Archivematica to be able to carry out preservation tasks, the
+Archivematica user needs to have execute privileges for directories and read
+permissions for files in the transfer source location.
```
Comment (ross-spencer) [inline user-manual/transfer/transfer.rst:137]: This is good Sara. An obvious trip-up for a lot of folk. 

I wonder if it worth having some snippets here? I think Ops will have some shortcuts I don't but, could we go as far as to say, `0744` for  directories, `0644` for files (I think those are correct reading the instructions here), and show the `chmod` example? Maybe too `chown` for the AM user? 

It would be great to have for our reference too :) 

Did you ever see this [permissions calculator](http://permissions-calculator.org/) btw? I look it up all the time because I never remember the numbers, but maybe also useful to the reader here?

**Review by ross-spencer — COMMENTED** [review]
Just left the one comment, this is a really great addition to have.

File: user-manual/transfer/transfer.rst (hunk 1 above)
Comment (sallain) [reply user-manual/transfer/transfer.rst:137]: I originally wanted to add more specific instructions - but I spent about 90 minutes testing different combinations of permissions and there wasn't really a clear-cut minimum threshold of permissibility. So I decided to go broad!

Comment (ross-spencer) [reply user-manual/transfer/transfer.rst:137]: I hear you. We had that tricky ticket in 1.11 where it wasn't working at all, but it wasn't clear what was the right octal :) I wonder if a middle-ground might be to say, _YMMV but these work for us? e.g. on an ArchivesDirect install?_ - I see Miguel is penciled for review, so, maybe he can contribute to that? Otherwise, I appreciate the approach and going broad so that we have something and not nothing.

Comment (sromkey) [reply user-manual/transfer/transfer.rst:137]: I personally favour the broad approach, I suspect it makes enough sense to sys admins, and if it didn't for some reason they can reach out on the user forum, etc.

**Review by sromkey — APPROVED** [review]
